x/gov/client/rest/grpc_query_test.go

```diff
@@ -119,23 +119,33 @@ func (s *IntegrationTestSuite) TestGetProposalsGRPC() {
 	val := s.network.Validators[0]
 
 	testCases := []struct {
-		name    string
-		url     string
-		headers map[string]string
-		expErr  bool
+		name             string
+		url              string
+		headers          map[string]string
+		wantNumProposals int
+		expErr           bool
 	}{
 		{
 			"get proposals with height 1",
 			fmt.Sprintf("%s/cosmos/gov/v1beta1/proposals", val.APIAddress),
 			map[string]string{
 				grpctypes.GRPCBlockHeightHeader: "1",
 			},
+			0,
 			true,
 		},
 		{
 			"valid request",
 			fmt.Sprintf("%s/cosmos/gov/v1beta1/proposals", val.APIAddress),
 			map[string]string{},
+			3,
+			false,
+		},
+		{
+			"valid request with filter by status",
+			fmt.Sprintf("%s/cosmos/gov/v1beta1/proposals?proposal_status=1", val.APIAddress),
```

Comment:
gRPC queries & grpc-gateway should not be using int versions of enum values, but rather the string values of those proto enums.

We should probably do something similar to how we handle `BondStatus` enums in `/staking/validators` queries [see here](https://github.com/cosmos/cosmos-sdk/blob/master/x/staking/keeper/querier.go#L79-L83).